src/App.tsx

```diff
@@ -1,9 +1,188 @@
+import { Navigate, Route, Routes } from 'react-router-dom';
 import './App.scss';
+import './utils/normalize.scss';
+import { useState, useEffect } from 'react';
+import { HeadNavigation } from './components/HeadNavigation/HeadNavigation';
+import { HomePage } from './pages/HomePage/HomePage';
+import { NotFound } from './pages/NotFoundPage/NotFound';
+import { PhonesPage } from './pages/PhonesPage/PhonesPage';
+import { TabletsPage } from './pages/TabletsPage/TabletsPage';
+import { AccessoriesPage } from './pages/AccessoriesPage/AccessoriesPage';
+import { Phone } from './types/Phone';
+import { getPhones } from './helpers/fetchPhones';
+import { Loader } from './components/Loader/Loader';
+import { FootNavigation } from './components/FootNavigation/FootNavigation';
+import { PhoneDetails } from './pages/PhoneDetailsPage/PhoneDetails';
+import { Favourites } from './pages/FavouritesPage/Favourites';
+import { Cart } from './pages/CartPage/Cart';
+import { CartItem } from './types/CartItem';
 
-const App = () => (
-  <div className="App">
-    <h1>React Phone Catalog</h1>
-  </div>
-);
+const App = () => {
+  const [phones, setPhones] = useState<Phone[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [searchResults, setSearchResults] = useState(phones);
+  const [searchQuery, setSearchQuery] = useState('');
+  const [likedProducts, setLikedProducts] = useState<Phone[]>([]);
+  const [cartProducts, setCartProducts] = useState<CartItem[]>([]);
```

Comment:
I'd recommend you create `useLocalStorage` custom hook to handle values stored in the local storage instead of saving them here and passing them as props to child components
you may google it or ask in the chat if you need help